tls/s2n_server_new_session_ticket.c

```diff
@@ -74,3 +75,75 @@ int s2n_server_nst_send(struct s2n_connection *conn)
 
     return 0;
 }
+
+int s2n_tls13_server_nst_send(struct s2n_connection *conn)
+{
+    notnull_check(conn);
+
+    /* Write message type because session resumption in TLS13 is a post-handshake message */
+    GUARD(s2n_stuffer_write_uint8(&conn->handshake.io, TLS_SERVER_NEW_SESSION_TICKET));
```

Comment:
Does the handler write the type and size for post-handshake handshake messages but not for normal handshake messages? That seems like something we should change, although I'm not sure it's in the scope of this PR.

Reply:
I'll make an issue for this.

Reply:
#2592